### PR TITLE
Ultimate Flounce Level Complete

### LINKED ISSUE #677: Leo removed from team page

### PR DESCRIPTION
**Pull request by nikeshashar**
@pitchinvasion one for you to merge tomorrow & closes #677

### CODE REVIEW

File: data/team.yml [hunk 1]
```diff
@@ -43,11 +43,6 @@
   job_title: Gatekeeper
   about: Nikesh is the self proclaimed gatekeeper for Makers Academy. After completing the course in June 2014, he refused to leave so we gave him a job. He now spends time selecting the best students for each cohort, facilitates the PreCourse and works with engineering to automate operations at Makers Academy. He also part of the Escape the City Founding Members and runs a charity in his “down” time.
```
Comment (roidriscoll) [inline data/team.yml]: image

**Comment by pitchinvasion** [comment]
I don't think I can top that gif - diva out 

🎤  💧

**Comment by Arfah** [comment]
http://giphy.com/gifs/bye-SuWKzYEuEbQAg

**Comment by Arfah** [comment]
giphy

**Pull request merged**